Assets/Scripts/States/Enemy/EnemyAttackState.cs

```diff
@@ -0,0 +1,27 @@
+
+using Tanks.tank;
+using UnityEngine;
+
+public class EnemyAttackState : EnemyStates
```

Comment:
If you have created different states , then all relevant methods should be inside that particular state. For eg. Chase state should have chase method , or should be implemented in controller class.